domain-models-runtime/src/main/java/org/folio/rest/RestRouting.java

```diff
@@ -433,7 +433,9 @@ private static void parseParams(RoutingContext rc, Buffer body, Iterator<Map.Ent
   static void invoke(Method method, Object[] params, Object o, RoutingContext rc,
                      Map<String, String> headers, Handler<AsyncResult<Response>> resultHandler) {
 
-    headers.forEach(FolioLoggingContext::put);
+    FolioLoggingContext.put(FolioLoggingContext.TENANT_ID_LOGGING_VAR_NAME, headers.get(RestVerticle.OKAPI_HEADER_TENANT));
+    FolioLoggingContext.put(FolioLoggingContext.REQUEST_ID_LOGGING_VAR_NAME, headers.get(RestVerticle.OKAPI_REQUESTID_HEADER));
+    FolioLoggingContext.put(FolioLoggingContext.USER_ID_LOGGING_VAR_NAME, headers.get(RestVerticle.OKAPI_USERID_HEADER));
```

Comment:
Can you also put MODULE_ID_LOGGING_VAR_NAME for https://github.com/folio-org/raml-module-builder/blob/v33.2.2/domain-models-runtime/src/main/resources/log4j2.properties#L15 ? org.folio.rest.tools.utils.ModuleName provides the name.

Reply:
ok